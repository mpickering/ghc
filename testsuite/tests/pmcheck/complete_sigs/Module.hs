module Module where

import ModuleA

foo :: A -> Bool
foo B = True


data D = D | E

{-# COMPLETE D #-}

foo2 :: D -> Bool
foo2 D = False
