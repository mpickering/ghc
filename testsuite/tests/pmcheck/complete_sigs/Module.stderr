[1 of 2] Compiling ModuleA          (.hs -> .o)
[2 of 2] Compiling Module           (.hs -> .o)
